js/src/components/shipping-rate-section/minimum-order-card/getMinimumOrderHandlers.test.js

```diff
@@ -0,0 +1,290 @@
+/**
+ * Internal dependencies
+ */
+import getMinimumOrderHandlers from './getMinimumOrderHandlers';
+
+describe( 'getMinimumOrderHandlers', () => {
+	let mockOnChange;
+	let handlers;
+
+	beforeEach( () => {
+		const value = [
+			{
+				id: '1',
+				country: 'US',
+				method: 'flat_rate',
+				currency: 'USD',
+				rate: 20,
+				options: {},
+			},
+			{
+				id: '2',
+				country: 'AU',
+				method: 'flat_rate',
+				currency: 'USD',
+				rate: 20,
+				options: {
+					free_shipping_threshold: 50,
+				},
+			},
+			{
+				id: '3',
+				country: 'CN',
+				method: 'flat_rate',
+				currency: 'USD',
+				rate: 25,
+				options: {
+					free_shipping_threshold: 50,
+				},
+			},
+		];
+
+		mockOnChange = jest.fn();
+		handlers = getMinimumOrderHandlers( {
+			value,
+			onChange: mockOnChange,
+		} );
+	} );
+
+	describe( 'handleAddSubmit', () => {
+		it( 'returns value with the newly added group', () => {
```

Comment:
:construction: Same as in https://github.com/woocommerce/google-listings-and-ads/pull/1424#discussion_r860274770

```suggestion
		it( 'Calls the `onChange` callback with the value containing the newly added group', () => {
```